app/src/main/java/org/mozilla/focus/fragment/AboutFragment.kt

```diff
@@ -0,0 +1,182 @@
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.Start
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import kotlinx.coroutines.Job
+import mozilla.components.browser.state.state.SessionState
+import org.mozilla.focus.R
+import org.mozilla.focus.databinding.FragmentAboutBinding
+import org.mozilla.focus.ext.components
+import org.mozilla.focus.settings.BaseSettingsLikeFragment
+import org.mozilla.focus.state.AppAction
+import org.mozilla.focus.utils.SupportUtils.manifestoURL
+import org.mozilla.geckoview.BuildConfig
+
+class AboutFragment : BaseSettingsLikeFragment() {
+
+    private val openLearnMore = {
+        val tabId = requireContext().components.tabsUseCases.addTab(
+            url = manifestoURL,
+            source = SessionState.Source.Internal.Menu,
+            selectTab = true,
+            private = true
+        )
+        requireContext().components.appStore.dispatch(AppAction.OpenTab(tabId))
+    }
+
+    override fun onResume() {
+        super.onResume()
+        updateTitle("About")
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        val binding = FragmentAboutBinding.inflate(inflater, container, false)
+
+        val appName = requireContext().resources.getString(R.string.app_name)
+        val aboutContent =
+            requireContext().getString(R.string.about_content, appName, "")
+
+        val learnMore = aboutContent
+            .substringAfter("<a href=>")
+            .substringBefore("</a></p>")
+
+        val content =
+            aboutContent
+                .replace("<li>", "\u2022 \u0009 ")
+                .replace("</li>", "\n")
+                .replace("<ul>", "\n \n")
+                .replace("</ul>", "")
+                .replace("<p>", "\n")
+                .replace("</p>", "")
+                .replaceAfter("<br/>", "")
+                .replace("<br/>", "")
```

Comment:
This feels a bit hacky, given that we can't really control the number of line breaks in the translations. Maybe we should consider splitting the strings. Especially if it also allows us to improve the layout. And it will confuse translators less...

Maybe we can ask @Delphine if that would be okay, I'll approve here anyways since we could try this for now (but we should look at some of the translations to see if that splitting approach works)

Reply:
Yes, this was my initial proposal, but it has been postponed until after MVP when we would eventually even change the text on this screen.